nntrainer/graph/network_graph.cpp

```diff
@@ -636,6 +636,8 @@ std::vector<TensorDim> NetworkGraph::getOutputDimension() const {
 std::vector<std::shared_ptr<Layer>>
 NetworkGraph::getUnsortedLayers(const std::string &input_layer,
                                 const std::string &output_layer) const {
+  /// @FIXME: this won't work if input, output layers are not in order
```

Comment:
Yes, getMergeableGraph must use getSortedLayers after sorting the graph.

Reply:
Further, `getUnsortedLayers` must be removed. There should be rather `getAllNames` and `getLayerByName` instead of `getUnsortedLayers`.

Reply:
Updating the comment here